### DISCUSSION TITLE
How can i change the layout of the page to rearrange cards?

### DISCUSSION THREAD

**Discussion by junioraze** [h2oai/wave#2306] [Q&A] [answered]
The is any way to change the layout of the page based on some event? I know in docs we have a mention about the responsive change based on the breakpoint of the device. The is a way to force some change like that one? 

Plus+ i cant find a good docummentation about `ui.boxes` how i know who box link with who layout?

**Comment by mturoci** [comment, marked as answer]
See https://wave.h2o.ai/docs/layout/. As for changing the layout, you need to either pick grid or flex layout and stick with your choice.
